web/themes/custom/cloudy/src/css/_elements.scss

```diff
@@ -21,3 +21,46 @@ a > i[class^="fab"] {
     color: $cloudy-color-purple-900;
   }
 }
+
+// Add links to Expand or Collapse all
+.aria-accordion a.toggle-accordion::before {
```

Comment:
I was just working on reviewing the WCAG 2.1 AA requirements that we need to meet and saw this one: https://www.w3.org/WAI/WCAG22/Techniques/failures/F87.html

I know you mentioned there as some uncertainty about the accessibility implications of using `::before` and `::after`, but it looks like it would be considered a failure according to that link.

Reply:
Looks like the PR was only merged in June while the idea was proposed in 2022. Will be interesting to see how the legal system deal with quickly evolving standard and technologies.
https://github.com/w3c/wcag/pull/3818